source/test/TestApp/AutoGenTests/AutoGenAdminTestData.cpp

```diff
@@ -0,0 +1,1415 @@
+#include "TestAppPch.h"
+#include "TestContext.h"
+#include "TestApp.h"
+#include "AutoGenAdminTests.h"
+#include "XAsyncHelper.h"
+#include <playfab/PlayFabClientAuthApi.h>
+#include <playfab/PlayFabClientApi.h>
+#include <playfab/PlayFabProfilesApi.h>
+#include <playfab/PlayFabAdminApi.h>
+#include <playfab/PlayFabAuthenticationAuthApi.h>
+#include <playfab/PlayFabClientDataModels.h>
+
+namespace PlayFabUnit
+{
+
+void AutoGenAdminTests::FillPlayFabAdminAbortTaskInstanceRequest( PlayFab::AdminModels::AbortTaskInstanceRequest* request )
+{
+    // TODO: debug  test
+    PlayFab::JsonDocument inputJson;
+    inputJson.Parse( "{ \"TaskInstanceId\": \"E52531B47B1761C7-4\"}" );
+    request->FromJson(inputJson);
+}
+
+void AutoGenAdminTests::FillPlayFabAdminAddLocalizedNewsRequest( PlayFab::AdminModels::AddLocalizedNewsRequest* request )
+{
+    // TODO: debug  test
+    PlayFab::JsonDocument inputJson;
+    inputJson.Parse( "{ \"NewsId\": \"74623b12-6c80-ee4b-7c3b-58e638aa62bd\", \"Language\": \"en\", \"Title\": \"News of the Day\", \"Body\": \"<b>We have a new object to buy!</b>\"}" );
```

Comment:
Would it be worth making it clear what is changed manually compared to what came from the swagger file?

Reply:
how would you do that?

Reply:
see my other on the other thread